website/js/components/insilicopred/SilicoPredTile.js

```diff
@@ -0,0 +1,167 @@
+/*eslint-env browser */
+/*global require: false, module */
+'use strict';
+
+import React from "react";
+import mapValues from 'lodash/mapValues';
+import maxBy from 'lodash/maxBy';
+
+import CollapsibleTile from "../collapsible/CollapsibleTile";
+import CollapsibleSection from "../collapsible/CollapsibleSection";
+
+import SplicingLevelSubtile from "./SplicingLevelSubtile";
+import ProteinLevelSubtile from "./ProteinLevelSubtile";
+import InSilicoPredSubtile from "./InSilicoPredSubtile";
+import {isNumeric} from "../../util";
+
+function extractSplicePayload(data, useTranscriptSplicePos) {
+    return {
+        donor: {
+            prior: data.refDonorPrior,
+            wild: {
+                sequence: data.refRefDonorSeq,
+                MES: data.refRefDonorMES,
+                zScore: data.refRefDonorZ
+            },
+            variant: {
+                sequence: data.altRefDonorSeq,
+                MES: data.altRefDonorMES,
+                zScore: data.altRefDonorZ
+            }
+        },
+        denovo: {
+            prior: data.deNovoDonorPrior,
+            wild: {
+                sequence: data.refDeNovoDonorSeq,
+                MES: data.refDeNovoDonorMES,
+                zScore: data.refDeNovoDonorZ,
+                donorPosition: useTranscriptSplicePos ? data.deNovoDonorTranscriptSplicePos : data.deNovoDonorGenomicSplicePos
+            },
+            variant: {
+                sequence: data.altDeNovoDonorSeq,
+                MES: data.altDeNovoDonorMES,
+                zScore: data.altDeNovoDonorZ,
+                donorPosition: useTranscriptSplicePos ? data.deNovoDonorTranscriptSplicePos : data.deNovoDonorGenomicSplicePos
+            },
+            closest: {
+                sequence: data.closestDonorRefSeq,
+                MES: data.closestDonorRefMES,
+                zScore: data.closestDonorRefZ,
+                donorPosition: useTranscriptSplicePos ? data.closestDonorTranscriptSplicePos : data.closestDonorGenomicSplicePos
+            },
+            closestAlt: {
+                sequence: data.closestDonorAltSeq,
+                MES: data.closestDonorAltMES, // where is this?
```

Comment:
?

Reply:
Ah, that's a cruft comment. The field was mentioned in the wireframes, but wasn't included in Amy's initial version of the document listing the ISPP fields; it's since been added, and it's in the output.